Documentation/contributors/design-proposals/operator-validation-library.md

```diff
@@ -53,3 +53,191 @@ There is also a Command Line Tool that serves as an example of how a user can in
   - Ideas: include a version of the library alongside the CSV type definition so that it is versioned in the same way the CSV is versioned.
+
+# Fake README
+
+## Installing
+
+### Command Line Tool
+
+You can interact with this library with a command line tool. 
+
+You can install the `operator-verify` tool from source using:
+
+```
+$ go install
+```
+
+```bash
+$ echo $PATH
+```
+
+If you do not have your workspace's `bin` subdirectory in your `$PATH`, 
+
+```bash
+$ export PATH=$PATH:$(go env GOPATH)/bin
+```
+
+This adds your workspace's bin subdirectory to your PATH. As a result, you can use the `operator-verify` tool anywhere on your system. Otherwise, you would have to `cd` to your workspace's `bin` directory to run the executable. Verify that we can find `operator-verify` in our new `PATH`:
+
+```bash
+$ which operator-verify
+```
+
+This should return something like:
+
+```
+~/go/bin/operator-verify
+```
+
+To verify that the library installed correctly, use it to validate your ClusterServiceVersion yaml,
+
+```
+$ operator-verify csv /path/to/filename.yaml
+```
+
+## Usage
+
+The command line tool's `help` command gives the following output:
+
+```text
+operator-verify is a command line tool for the Operator Manifest Verification Library. This library provides functions to validate the operator manifest bundles against Operator-Lifecycle-Manager's clusterServiceVersion type, customResourceDefinitions, and package yamls. Currently, this application supports static validation of both individual operator files and the manifest bundle. 
+
+Usage:
+  operator-verify [flags]
+  operator-verify [command]
+
+Available Commands:
+  help        Help about any command
+  csv         Validate CSV against OLM's type
+  crd         Validate manifest CRDs
+  package     Validate package yaml
+  manifest    Validate operator manifest bundle  
+
+Flags:
+  -h, --help    help for operator-verify
+  -i, --ignore  ignore warnings in log
+
+Use "operator-verify [command] --help" for more information about a command.
+```
+
+
+### Commands 
+**For individual files**: 
+
+```
+$ operator-verify csv /path/to/csv.yaml
+```
+
+Validates the given CSV yaml file against OLM type and reports errors and warnings as described in the `List of Errors` section below.
+
+Similarly, 
+
+```
+$ operator-verify crd /path/to/crd.yaml
+```
+
+and 
+
+```
+$ operator-verify package /path/to/package.yaml
+```
+
+validates CRD and package yaml, respectively. The `crd` command can accept and validate both a single yaml file and a bunch of CRDs at once and report a separate log for each.
+
+**For manifest bundle**:
+
+```
+$ operator-verify manifest /path/to/bundle
+```
+
+Here `path/to/bundle` is a directory structure as per [manifest format](https://github.com/operator-framework/operator-registry#manifest-format). It can be either be flattened or nested with different versions of yaml files. This command reports errors and/or warnings for each file in the bundle.
```

Comment:
Nested is the only supported format going forward. I wouldn't even mention "nested" just call it **the** operator manifest format (with the link you have above).

Reply:
Done.